src/%ZPM/PackageManager/Developer/Processor/Default/Document.cls

```diff
@@ -399,8 +400,40 @@ Method OnLoad(pFullResourcePath As %String, pVerbose As %Boolean, pCompile As %B
       Set pLoadedList = ""
       Set tSC = $$Import^%apiRTN(pFullResourcePath,"replace,compile="_pCompile,,,.tResult,$Select(pVerbose:"d",1:"-d"))
     } Else {
-      Set tSC = $System.OBJ.Load(pFullResourcePath,tFlags,,.pLoadedList)
-      $$$ThrowOnError(tSC)
+      Set tProcessed = 0
+      set fileExt = $$$UPPER($Piece(pFullResourcePath, ".", *))
+      if '$LISTFIND($LB("CLS", "MAC"), fileExt), ##class(%RoutineMgr).UserType(..ResourceReference.Name, .docclass, .doctype) {
```

Comment:
Why do we need to check fileExt? UserType should return 0 for both of those.

Reply:
ResourceReference.Name and pFullResourcePath are different
we have a case with `bpl`, and it can be as `cls` file

Reply:
What about DFI?

Reply:
This one in the test at the moment
https://github.com/intersystems/ipm/blob/87c5bf7e7d/tests/integration_tests/Test/PM/Integration/_data/resource-test/dfi/Sample_Operational_Reports/Auditing_Overview.dashboard.xml